core/modules/simpletest/backdrop_web_test_case_cache.php

```diff
@@ -159,7 +159,9 @@ protected function tearDown() {
   /**
    * Alter tables to MyISAM engine to speed up tests.
    *
-   * MyISAM is faster to delete and copy tables. It gives small adventage when /var/lib/mysql on SHM (memory) device, but much bigger when tests run on regular device.
+   * MyISAM is faster to delete and copy tables. It gives small advantage when
+   * /var/lib/mysql on SHM (memory) device, but much bigger when tests run on
+   * regular device.
```

Comment:
```suggestion
   * MyISAM is faster to delete and copy tables. Its adventage is small when 
   * using /var/lib/mysql on a SHM (memory) device but is much bigger when tests 
   * are run on a regular device.
```

Reply:
@bugfolder I think the point of this issue is to remove typos, not add more... :wink: 

> Its *adventage* is...

Reply:
Doi!